src/constants/url.ts

```diff
@@ -24,6 +24,7 @@ export const ALL_TAGS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tags`;
 export const ALL_LEVELS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/levels`;
 export const ITEM_TYPES = { task: 'TASK' };
 export const TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks`;
+export const TASK_REQUEST_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/taskRequests`;
```

Comment:
- add a to-do comment if we are planning to change the API's name.

Reply:
renamed it to `DEFAULT_TASK_PRIORITY`

Reply:
Done.